c++/src/kj/exception.c++

```diff
@@ -733,6 +733,10 @@ String KJ_STRINGIFY(const Exception& e) {
              stringifyStackTrace(e.getStackTrace()));
 }
 
+StringPtr KJ_STRINGIFY(CanceledException e) {
```

Comment:
Eh, I don't think a stringifier is needed for this type, since it has only one possible value.